presto-parquet/src/main/java/com/facebook/presto/parquet/dictionary/DoubleDictionary.java

```diff
@@ -31,7 +33,8 @@ public DoubleDictionary(DictionaryPage dictionaryPage)
         super(dictionaryPage.getEncoding());
         content = new double[dictionaryPage.getDictionarySize()];
         DoublePlainValuesReader doubleReader = new DoublePlainValuesReader();
-        doubleReader.initFromPage(dictionaryPage.getDictionarySize(), dictionaryPage.getSlice().getBytes(), 0);
+        final ByteBufferInputStream in = ByteBufferInputStream.wrap(ByteBuffer.wrap(dictionaryPage.getSlice().getBytes()));
```

Comment:
- `in` -> `inputStream`
- we don't use final local variables, please remove final modifier.